### ISSUE TITLE
i installed the internet archive 16 and 64 gb one and it goes to a black screen

### ISSUE THREAD

**Issue by noor8706** [MrJackSpade/RG35XXP-XFCE#7]
when i boot the 16 and 64 gb image into my sd card it shows ambernic and goes black tho the teck toy tinker xfce works tho i want the ambernic roms menu where i can open games :Update i think its in sleep mode tho when i hold it i can see the menu tho it shuts off instantly

**Comment by Mark-B97** [comment]
I also had no success. Mine doesn't even show the anbernic logo nor anything, the screen just turns on and it's stuck with black screen

**Comment by MrJackSpade** [comment]
You flashed the SD card image?

Did you flash it to the internal, or external card?

**Comment by Mark-B97** [comment]
I don't know what happened the first time but I did it again and it worked? I flashed with dd, maybe it was the block size setting that I changed but theoretically that shouldn't affect the result or the flashed image in any way. It works great, I changed the font settings to enable hinting and it makes text way more readable. It's a bit slow sometimes and that's to be expected but I wonder if lxqt would perform better since it's such a limited device and lxqt is the least demanding DE.
